community/pike/APKBUILD

```diff
@@ -0,0 +1,188 @@
+# Maintainer: Paweł Tomak <pawel@tomak.eu>
+pkgname=pike
+_pkgname=Pike
+pkgver=8.0.610
+pkgrel=0
+pkgdesc="Pike Programing language"
+url="https://pike.lysator.liu.se"
+arch="x86_64"
+license="GPL-2.0-or-later"
+options="!check" # there are no tests for those packages
+depends="gmp nettle nettle-utils readline"
```

Comment:
`gmp` (`so:libgmp.so.10`) and `nettle` (`so:libnettle.so.6`) can be removed from this `depends` list